gitspy/RepositoryCollectionViewCell.swift

```diff
@@ -10,7 +10,10 @@ import UIKit
 
 class RepositoryCollectionViewCell: UICollectionViewCell {
     
-    @IBOutlet weak var repositoryTitleLabel: UILabel!
+    @IBOutlet private weak var repositoryTitleLabel: UILabel!
     
+    func setRepository(repository: Repository?) {
+        repositoryTitleLabel.text = repository?.name
+    }
```

Comment:
For now I would just create a method for setting the label's text or expose the uilabel as it was before. But later when this gets bigger we could think about something like this https://github.com/joanromano/SCTestClient/tree/master/SCTestClient/Cell%20Adapters
What do you think?